Sources/BranchSDK/BNCServerRequestQueue.m

```diff
@@ -287,6 +287,8 @@ - (void)retrieve {
                 }
             }
             self.queue = decodedQueue;
+            // Requests are loaded into queue now. Delete queue file stored on disk.
+            [self removeSaveFile];
```

Comment:
Just confirming this operation is atomic

Reply:
Yes it is.